### ISSUE TITLE
Missing dijit/Editor icons for some plugins

### ISSUE THREAD

**Issue by comxd** [dojo/dijit-themes#34]
- FindReplace (extraPlugins)
- Blockquote (extraPlugins)
- unlink
- subscript
- superscript
- insertHorizontalRule

![image](https://cloud.githubusercontent.com/assets/9022239/22620713/1d5d58d8-eb12-11e6-951e-f031cc6cf7cd.png)
![image](https://cloud.githubusercontent.com/assets/9022239/22620736/8607a1fe-eb12-11e6-8d0b-d24d2308a5c1.png)
![image](https://cloud.githubusercontent.com/assets/9022239/22621023/f7895140-eb19-11e6-9972-aa06047a5b0a.png)

**Comment by ThibaultJanBeyer** [comment]
yes, what about the icons? Is there any roadmap on when they will come out? Thanks
